app/controllers/OtherTaxesController.scala

```diff
@@ -17,7 +17,7 @@
 package controllers
 
 import config.FrontendAppConfig
-import config.featureToggles.FeatureSwitch.{ECLSwitch, NewADPJourney, Pillar2Switch, PptSwitch}
+import config.featureToggles.FeatureSwitch.{ECLSwitch, Pillar2Switch}
```

Comment:
I strongly request you to have a short call with @shankaran-hmrc  tomorrow reagrding this befor merge. He spent good amount of time for Switches, he can help you if any repos need this change with respect to this. I just skimmed through the code. All looks good to me.